### PR TITLE
Issue #4845: Removed checkstyle dependencies from illegalimport

### PR DESCRIPTION
**Pull request by Guneetsinghtuli**
Part of #4845

### CODE REVIEW

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/InputIllegalImportDefault1.java [hunk 1]
```diff
@@ -9,13 +9,10 @@
 
 package com.puppycrawl.tools.checkstyle.checks.imports.illegalimport;
 
-import com.puppycrawl.tools.checkstyle.checks.imports.illegalimport.*;
-
 import java.io.*; // violation
 import java.lang.*;
 import java.sql.Connection;
 import java.util.List;
-import java.util.List;
```
[the remark this answers is not in the document]
Comment (Guneetsinghtuli) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/InputIllegalImportDefault1.java:18]: Okay will do that, Just wanted to know the purpose of why we are having 2 import for `java.util.List`

[the remark this answers is not in the document]
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/InputIllegalImportDefault1.java:18]: Tests are weird on purpose. Checkstyle should work all compile able code, even this code is nonsense.

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**